README.md

```diff
@@ -76,3 +76,33 @@ To compile MO files and update live translations
 $ django-admin.py compilemessages ;
 $ for app in {main,viewer} ; do cd kobocat/apps/${app} && django-admin.py compilemessages && cd - ; done
 ```
+## Testing in Kobocat
+
+For Kobo-Install Users, enter the folder for Kobo-Install and run this command
```

Comment:
I think the correct way is `KoBoCAT` (see https://github.com/kobotoolbox/kobocat/blob/master/README.md) 
I used to write KoBo-Install, but we decided to change to `kobo-install` (https://github.com/kobotoolbox/kobo-install/commit/b4da7d7415af60ab2c299f1a4aeb011b76b5dcaa) to match with `kobo-docker`

It's not an error at all but I think it's better to be consistent in every repo. What do you think?